rust/aiconfigurator-core/src/perf.rs

```
@@ -0,0 +1,816 @@
+
+        for point in self.context_attention.iter().filter(|point| {
+            point.fmha_quant == fmha_quant
+                && point.kv_cache_quant == kv_cache_quant
+                && point.num_kv_heads == query_kv_heads
+                && point.head_dim == head_dim
+                && point.window_size == 0
+        }) {
+            let score = relative_distance(point.batch_size, batch_size)
+                + relative_distance(point.sequence_tokens, full_sequence_tokens)
+                + relative_distance(point.num_heads, num_heads);
+            if best.map_or(true, |(best_score, _)| score < best_score) {
```

Clippy (1.95.0)

[warning] this `map_or` can be simplified

(warning)